indigo/distribution.yaml

```diff
@@ -9137,16 +9137,16 @@ repositories:
     doc:
       type: git
       url: https://github.com/baalexander/rospy_message_converter.git
-      version: indigo-devel
+      version: master
     release:
       tags:
         release: release/indigo/{package}/{version}
-      url: https://github.com/jihoonl/rospy_message_converter-release.git
-      version: 0.3.0-2
+      url: https://github.com/baalexander/rospy_message_converter-release.git
```

Comment:
The baalexander/ release repo is a new repo as I was getting an error with bloom-release. I can try forking the jihoonl/ release repo and re-running bloom-release if that's preferable.